src/main/java/core/basesyntax/services/LineParser.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.services;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class LineParser {
+    public static final int CORRECT_ARG_AMOUNT = 4;
+    public static final int QUANTITY_INDEX = 2;
+    public static final int DATE_INDEX = 3;
+
+    public String[] parse(String line) {
+        String[] splitedLine = line.split(",");
+        if (splitedLine.length != CORRECT_ARG_AMOUNT) {
+            throw new RuntimeException("Wrong argument format!");
+        }
+        try {
+            Integer.parseInt(splitedLine[QUANTITY_INDEX]);
+            LocalDate.parse(splitedLine[DATE_INDEX]);
```

Comment:
That's optional, but these two lines responsible not for parsing, but for validation(meaning if you remove them, that won't influence output in case of valid input). So more logical would be to parse line into model "record" with fields like type, quantity, etc and return initialized object from method. In that case method naming will fully correspond with its purpose.

Reply:
okay, I ve got it, but here I think nothing bad happen if I will not change this

Reply:
that's up to you